packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -102,7 +104,13 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
       return;
     }
 
-    if (prevState.isStickyTop !== this.state.isStickyTop || prevState.isStickyBottom !== this.state.isStickyBottom) {
+    const currNonStickyContentOffsetHeight = this.nonStickyContent ? this.nonStickyContent.offsetHeight : undefined;
+    if (
+      prevState.isStickyTop !== this.state.isStickyTop ||
+      prevState.isStickyBottom !== this.state.isStickyBottom ||
+      this.prevNonStickyContentOffsetHeight !== currNonStickyContentOffsetHeight
+    ) {
+      this._updatePrevNonStickyContentOffsetHeight(currNonStickyContentOffsetHeight ? currNonStickyContentOffsetHeight : 0);
```

Comment:
If the offsetHeight is 'undefined', why would we capture '0' as the offsetHeight?